apps/docs/src/app/core/component-docs/icon-tab-bar/examples/config-for-examples/icon-type-config.ts

```diff
@@ -0,0 +1,117 @@
+import { TabConfig } from '@fundamental-ngx/core/icon-tab-bar';
```

Comment:
I'm not sure that it will be working on the StackBlitz as "pure" files are not caught properly there. See my issues for more information https://github.com/SAP/fundamental-ngx/issues/5351